test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -47,13 +180,52 @@ winrt::ToastProgressData GetToastProgressData(std::wstring const& status, std::w
     return progressData;
 }
 
+bool ProgressResultOperationHelper(winrt::IAsyncOperation<winrt::ToastProgressResult> progressResultOperation, winrt::ToastProgressResult progressResult)
+{
+    if (progressResultOperation.wait_for(std::chrono::seconds(2)) != winrt::AsyncStatus::Completed)
+    {
+        progressResultOperation.Cancel();
+        return false; // timed out or failed
+    }
+
+    return progressResultOperation.GetResults() == progressResult;
+}
+
+bool PostToastHelper(std::wstring const& tag, std::wstring const& group)
+{
+    winrt::ToastNotification toast{ CreateToastNotification() };
+
+    toast.Tag(tag.c_str());
+    toast.Group(group.c_str());
+
+    winrt::ToastNotificationManager::Default().ShowToast(toast);
+
+    if (toast.ToastId() == 0)
+    {
+        return false;
+    }
+
+    return true;
+}
+
+bool BackgroundActivationTest() // Activating application for background test.
+{
+    return true;
+}
+
+bool UnregisterBackgroundActivationTest()
+{
+    winrt::ToastNotificationManager::Default().UnregisterActivator();
+    return true;
+}
+
 bool VerifyFailedRegisterActivatorUsingNullClsid()
 {
     try
     {
-        auto activationInfo = winrt::ToastActivationInfo::CreateFromActivationGuid(winrt::guid(GUID_NULL));
+        auto activationInfo = winrt::ToastActivationInfo::CreateFromActivationGuid(winrt::guid(GUID_NULL)); // The exception gets thrown here
```

Comment:
There are some changes since we got rid of assets. I would strongly recommend merging asap than continue iterating on this change.